### PR TITLE
newmac

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-11-25-new-macBook.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## New MacBook
+
+I just bought a MacBook a few years ago so I'm not interested in this year's macbook pro, but if I did not I would have surely bought it. It still has a lot of problems, but finally even at Apple stubborn as they are have realized that the buttlerfly keyboard was a complete and utter disaster.
```
Comment (MsSlacker) [inline _posts/2019-11-25-new-macBook.md]: A year ago. It's 2018 mbp...

Comment (MsSlacker) [reply _posts/2019-11-25-new-macBook.md]: Apple , (...) are ,

Comment (MsSlacker) [reply _posts/2019-11-25-new-macBook.md]: they have realised

Comment (MsSlacker) [reply _posts/2019-11-25-new-macBook.md]: buttlerfly - butterfly

File: _posts/2019-11-25-new-macBook.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+PROS
+
+- Finally a keyboard that actually works when you type
+It is astonishing that this needs to be in a list of pro for a laptop, but here we are. What a shame.
```
Comment (MsSlacker) [inline _posts/2019-11-25-new-macBook.md]: pro - > pros

File: _posts/2019-11-25-new-macBook.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+People say it is even better than the 2015 macbook pro keyboard and more in line with the magic keyboard, that's just great. I love the magic keyboard.
+
+- Finally the escape key is back
+Same as above.
+
+- The touch id and off button is now more visible. I took a week to realize where it was when I bought the 2018 macbook pro.
```
Comment (MsSlacker) [inline _posts/2019-11-25-new-macBook.md]: just say "when I bought mine "

File: _posts/2019-11-25-new-macBook.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Great specs
+You can future-proof your new macbook really well this year
+
+- Great microphone
+Finally we have a really good microphone already in our mac so during skype/slack/zoom calls we don't always have to use headphones + mic to get a decent audio. We will still need it to record a podcast or for soemthing where we want more sound fidelity, but for 99% of the times it will be good enough.
```
Comment (MsSlacker) [inline _posts/2019-11-25-new-macBook.md]: time is not countable...

File: _posts/2019-11-25-new-macBook.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+CONS
+
+- The Usual Prices
+As always prices are off the charts, but we're unfortunately all used to it.
+- Touchbar
+The touchbar is still here, it seems that Apple won't go back on this. It is way worse than having physical keys and the missclicks are a lot. I even had to hide a few buttons so that they didn't miss fire so often
```
Comment (MsSlacker) [inline _posts/2019-11-25-new-macBook.md]: change "a lot" to something else

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-11-25-new-macBook.md (hunk 1 above)
Comment (valeIT) [reply _posts/2019-11-25-new-macBook.md]: > realised


realized is ok.

**Review by MsSlacker — DISMISSED** [review]

**Comment by valeIT** [comment]
Can this be merged?

**Comment by valeIT** [comment]
> Can this be merged?

+1 :P

@MsSlacker

**Pull request merged**